rest_framework/locale/en_US/LC_MESSAGES/django.po

```diff
@@ -8,7 +8,7 @@ msgid ""
 msgstr ""
 "Project-Id-Version: PACKAGE VERSION\n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2016-07-12 16:13+0100\n"
+"POT-Creation-Date: 2019-04-11 23:21+0200\n"
```

Comment:
Translations are managed with transifex and are updated via the release process. These changes should be removed.

Reply:
Ok, changes in this file have been dropped